traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -33,52 +31,31 @@ import (
-		{1.2, http.MethodGet, "regions-wip.json$", wrapHeaders(wrapAuthWithData(regionsHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, ServersPrivLevel))},
-		{1.2, http.MethodGet, "regions-wip$", wrapHeaders(wrapAuthWithData(regionsHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, ServersPrivLevel))},
-		{1.2, http.MethodGet, "regions-wip.json$", wrapHeaders(wrapAuthWithData(serversHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, ServersPrivLevel))},
-		{1.2, http.MethodGet, "servers-wip.json$", wrapHeaders(wrapAuthWithData(serversHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, ServersPrivLevel))},
-		{1.2, http.MethodGet, "servers-wip$", wrapHeaders(wrapAuthWithData(serversHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, ServersPrivLevel))},
-		{1.2, http.MethodGet, "servers-wip.json$", wrapHeaders(wrapAuthWithData(serversHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, ServersPrivLevel))},
-		{1.2, http.MethodGet, "asns-wip$", wrapHeaders(wrapAuthWithData(ASNsHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, ServersPrivLevel))},
-		{1.2, http.MethodGet, "asns-wip.json$", wrapHeaders(wrapAuthWithData(ASNsHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, ServersPrivLevel))},
-		{1.2, http.MethodGet, "cdns-wip$", wrapHeaders(wrapAuthWithData(cdnsHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, CdnsPrivLevel))},
-		{1.2, http.MethodGet, "cdns-wip.json$", wrapHeaders(wrapAuthWithData(cdnsHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, CdnsPrivLevel))},
-		{1.2, http.MethodPost, "servers/{server}/deliveryservices$", wrapHeaders(wrapAuthWithData(assignDeliveryServicesToServerHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, PrivLevelOperations))},
-		{1.2, http.MethodGet, "divisions-wip$", wrapHeaders(wrapAuthWithData(divisionsHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, CdnsPrivLevel))},
-		{1.2, http.MethodGet, "divisions-wip.json$", wrapHeaders(wrapAuthWithData(divisionsHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, DivisionsPrivLevel))},
-		{1.2, http.MethodGet, "hwinfo-wip$", wrapHeaders(wrapAuthWithData(hwInfoHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, HWInfoPrivLevel))},
-		{1.2, http.MethodGet, "hwinfo-wip.json$", wrapHeaders(wrapAuthWithData(hwInfoHandler(d.DB), d.Insecure, d.Secrets[0], rd.PrivLevelStmt, HWInfoPrivLevel))},
-	}, rootHandler(d), nil
-}
-
-type RouteData struct {
-	PrivLevelStmt *sql.Stmt
-}
-
-func routeData(d ServerData) (RouteData, error) {
-	rd := RouteData{}
-	err := error(nil)
-
-	if rd.PrivLevelStmt, err = preparePrivLevelStmt(d.DB); err != nil {
-		return rd, fmt.Errorf("Error preparing db priv level query: ", err)
-	}
+	routes := []Route{}
+	//ASNs
+	routes = append(routes, Route{Version: 1.2, Method: http.MethodGet, Path: `asns-wip(\.json)?$`, Handler: ASNsHandler(d.DB), RequiredPrivLevel: ServersPrivLevel})
```

Comment:
Why make every route `append`?

```	routes = append(routes, Route{Version: 1.2, Method: http.MethodGet, Path: `cdns-wip(\.json)?$`, Handler: cdnsHandler(d.DB), RequiredPrivLevel: CdnsPrivLevel})``` 

Seems harder to read than 

```{1.2, http.MethodGet, `cdns-wip(\.json)?$`, cdnsHandler(d.DB), CdnsPrivLevel},```

Reply:
This was up for discussion and is to allow comments between the routes. I can go either way on it.

Reply:
I just tested
```    {1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring.json$", wrapHeaders(wrapAuth(monitoringHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},                                                                                                                                                                        // Regions                                                                                                                                                                                                                                                                                                                                      
    {1.2, http.MethodGet, "regions-wip/{id}$", wrapHeaders(wrapAuthWithData(regionsHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, ServersPrivLevel))},  ```
Compiles